### ISSUE TITLE
Feature: Create modules directly in acrticles

### ISSUE THREAD

**Issue by jenda77** [contao/core#8399]
I would like to have the possibility to create a new module directly in an article (this is posible in Typo3 and I think Contao could have it too). This would be also fine for redactors as well. Of course, there should be a limitation for members or users to forbid them to do it. 

I think the current way is too long and not really user-friendly for redactors. With this feature we could work more quick and could stay at one place where we want to do something with the content.

**Comment by aschempp** [comment]
There is an intended separation between modules and content elements. Modules are part of the page layout and should not be maintained by content editors, while content elements are "supposed" to be in the content, and not in the page layout.

**Comment by jenda77** [comment]
Yes, I know this current way. But on the other hand, it could be more easier for everyone who could edit it (sure, you could forbid this feature). All modules could be stored at the same place like now. But it could be very good to have the possibility to create a module directly in an article.

**Comment by Aybee** [comment]
:+1: 
I understand what @aschempp says but I think it's worth to think about it. Could be a handy feature.

Maybe this way:
![create-new-module-from-within-ce](https://cloud.githubusercontent.com/assets/1143013/16685847/2bc4c9e2-450e-11e6-8832-b59c1146e608.png)

**Comment by jenda77** [comment]
Yes. Then there would be 2 possibilities: for example, under Include Elements 1. to create a new modules and 2. to include existing one. It's just a suggestion. ;-)

**Comment by asaage** [comment]
I'm not a big fan of this.
It might be a nice helper for admins but it would require theme/module access for editors which is sth. i always try to avoid.
It would only make sense if we had content-type based permission settings which is unfortuately not the case.

**Comment by jenda77** [comment]
Yes, that´s another feature we need it. There is an extension from Andreas to manage the access but I think it should be a core feature.
